test_unstructured/chunking/test_base.py

```diff
@@ -398,6 +421,7 @@ def but_it_splits_its_table_into_TableChunks_when_the_table_text_exceeds_the_win
             "<tbody>\n"
             "<tr><td>Lo"
         )
+        assert not chunk.metadata.is_continuation
```

Comment:
These new assertions on `.is_continuation` are not related to this PR, just extra assurance for a couple PRs back where we added this behavior to `TextPreChunk`.